session.go

```diff
@@ -22,6 +22,8 @@ type ConnectOpts struct {
 	Database  string        `gorethink:"database,omitempty"`
 	AuthKey   string        `gorethink:"authkey,omitempty"`
 	Timeout   time.Duration `gorethink:"timeout,omitempty"`
+	SSL       bool          `gorethink:"ssl",omitempty"`
+	CaCert    string        `gorethink:"cacert",omitempty"`
```

Comment:
I think this field should be named `CACert` as I am trying to make the naming of functions and fields more "idiomatic" and ensuring that the driver passes golint+govet in preparation for v1.0.